assert/assert_array_includes_test.ts

```diff
@@ -29,6 +52,19 @@ missing: [
   {
     deno: "node",
   },
-]`,
-  );
+]
+`.trim(),
+    );
+  });
+
+  await t.step("Type-check failing cases", () => {
```

Comment:
```suggestion
  // https://github.com/denoland/deno_std/issues/3372
  await t.step("Type-check failing cases", () => {
```

It might be a good idea to provide a link explaining why this test was added.